pkg/operator/prestostore/query.go

```diff
@@ -78,6 +78,7 @@ func ImportFromTimeRange(logger logrus.FieldLogger, clock clock.Clock, promConn
 		})
 
 		promLogger.Debugf("querying Prometheus using range %s to %s", timeRange.Start, timeRange.End)
+		promLogger.Debugf("Prometheus query is: %s", cfg.PrometheusQuery)
```

Comment:
I like the idea of this debug statement, but it would be nicer if we were using a logger client that supported multiple levels of logging instead of info/debug/etc.

Reply:
Once I had the PromQL from the debug print, it was easy to track down in values.yaml with a grep. But without it I was pretty much at a loss. I think that would be case for a cluster admin who wanted to report a better bug or diagnose somewhat. Assuming they want less logs in a production cluster, tuning the operators to log above the debug level would be reasonable.